File: Atcoder/ABC/ABC165/d.py
def main():
    a, b, n = map(int, input().split())
    ans = 0
    for i in range(n, -1, -1):
        if ans == 9:
            break
        print((int(a * i / b) - a * int(i / b)))
        ans = max(ans, (int(a * i / b) - a * int(i / b)))
    print("-----"+str(ans)+"----")
if __name__ == '__main__':
    main()

File: Atcoder/ABC/ABC165/d.java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        long a = input.nextLong();
        long b = input.nextLong();
        long n = input.nextLong();
        long x = Math.min(b-1,n);
        long ans = (a*x/b) - a*(x/b);
        System.out.println(ans);
    }
}